scripts/generate-sitemap.mjs

```
@@ -0,0 +1,47 @@
+import { readdirSync, readFileSync, writeFileSync } from 'node:fs';
```

ESLint

[error] Member 'readFileSync' of the import declaration should be sorted alphabetically.

(sort-imports)